docs/source/en/using-diffusers/sdxl.md

```diff
@@ -96,16 +97,17 @@ image = pipeline(prompt=prompt).images[0]
 For image-to-image, SDXL works especially well with image sizes between 768x768 and 1024x1024. Pass an initial image, and a text prompt to condition the image with:
 
 ```py
-from diffusers import AutoPipelineForImg2Img
-from diffusers.utils import load_image
+from diffusers import AutoPipelineForImage2Image
+from diffusers.utils import load_image, make_image_grid
 
 # use from_pipe to avoid consuming additional memory when loading a checkpoint
 pipeline = AutoPipelineForImage2Image.from_pipe(pipeline_text2image).to("cuda")
-url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/sdxl-img2img.png"
 
-init_image = load_image(url).convert("RGB")
+url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/sdxl-img2img.png"
+init_image = load_image(url)
```

Comment:
Oops sorry, the initial image is the [output](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/sdxl-text2img.png) from the text-to-image pipeline in the previous section.

Reply:
Done.